packages/curve-ui-kit/src/shared/ui/ActionInfo.tsx

```diff
@@ -120,31 +120,31 @@ const ActionInfo = ({
           />
         )}
 
-        {/** Additional stack to add some space between left (icon), value and right (icon) */}
-        <Stack direction="row" alignItems="center" gap={Spacing.xxs}>
-          {valueLeft}
-
-          <WithSkeleton loading={!!loading}>
-            <Tooltip
-              title={valueTooltip}
-              placement="top"
-              slotProps={{
-                popper: {
-                  sx: {
-                    userSelect: 'none',
-                    pointerEvents: 'none',
-                  },
-                },
-              }}
-            >
+        <Tooltip
+          title={valueTooltip}
+          placement="top"
+          slotProps={{
+            popper: {
```

Comment:
This is already in `packages/curve-ui-kit/src/shared/ui/Tooltip.tsx` we should use that for style consistency